media/css/custom.scss

```diff
@@ -498,3 +498,20 @@ summary {
   height: 40px;
   padding: 8px;
 }
+
+.finance-container {
+  a {
+    float: left;
+    height: 168px;
+    margin: 16px;
+    text-align: center;
+    width: 228px;
+	&:hover {
```

Comment:
Rule declaration should be preceded by an empty line<br>Line should be indented with spaces, not tabs